CODE_OF_CONDUCT.md

```diff
@@ -21,6 +21,7 @@ Examples of unacceptable behavior by participants include:
 * Public or private harassment
 * Publishing others’ private information, such as a physical or electronic address, without explicit permission
 * Wasting other people’s time with low quality contributions, including but not limited to LLM and bot spam
+* Displaying politically charged imagery or content to provoke controversy
```

Comment:
```suggestion
* Displaying or referencing politically charged imagery or content
```
This would be more consistent with the rest of the bullet points. Which lack further clarification.

Reply:
This also seems to fall in line with the Contributor Covenant:

> “Everything is politics”, but the code of conduct is not about an advance of progressive/left-wing politics. It’s about establishing a minimal level of civil and professional collaboration. Civil, non-discriminatory, and professional behavior should be a baseline and shared value held by people of all ideologies, regardless of political affiliation (with the obvious exception of hate groups).